bitnami/contour/README.md

```diff
@@ -66,7 +66,7 @@ $ helm uninstall my-release
 | `fullnameOverride`   | String to fully override contour.fullname template                                     | `""`  |
 | `ingress.apiVersion` | Force Ingress API version (automatically detected if not set)                          | `""`  |
 | `kubeVersion`        | Force target Kubernetes version (using Helm capabilities if not set)                   | `""`  |
-
+| `extraDeploy`        | Array of extra objects to deploy with the release                                      | `[]`  |
```

Comment:
It seems you're manually editing the README.md to include the new parameters. Please note this is not required anymore. We developed a new tool called `readme-generator` that will automatically do it for you based on the metadata defined in the **values.yaml**:

- https://github.com/bitnami-labs/readme-generator-for-helm

Reply:
Yep, I already tried it, it not working, just got YAML validation error like:
'key over 1024 chars, line 135'

As YAML 100% correct,
so I just think readme-generator is broken and edit it manually

Reply:
That's so weird.. @miguelaeh could you take a look to this?

Reply:
Hi, it seems the action to generate the README is reporting missing metadata, you can check the output here: https://github.com/bitnami/charts/pull/7293/checks?check_run_id=3401616804

Reply:
`json-fields` broke readme generator

Reply:
I've regenerated README.md, thanks.